### PR TITLE
Add react-tagged-state

### PR DESCRIPTION
**Pull request by oleggrishechkin**
Hi, I'm added react-tagged-state

I'm also changed strange table markup

### CODE REVIEW

**Comment by dai-shi** [comment]
The result seems good. How did you accomplish this?

File: README.md [hunk 1]
```diff
@@ -442,7 +456,7 @@ in your browser, and click the button very quickly. (check the console log)
     <td>:x:</td>
   </tr>
 
-  </tr>
+  <tr>
```
Comment (dai-shi) [inline README.md:459]: Oh, no. I didn't realize it.

**Comment by oleggrishechkin** [comment]
> The result seems good. How did you accomplish this?

I changed `useTagged` to support Concurrent Mode (previous version was like `react-redux` - value saved in mutable ref object), now it's like `useSubscription` and value saved in state with selector function (I think `react-redux` can do the same approach)

Honestly, I don't understand how it's all works.

Thanks for your post about Concurrent Mode `and reactive-react-redux`, I cheched your implementation of `useMutableSource` in `valtio`, it's really help me.

**Comment by dai-shi** [comment]
https://github.com/oleggrishechkin/react-tagged-state/blob/47518abbd1b777f93a1b90cae122015b7d76ab07/src/index.ts#L124

Okay, it's async update like valtio. Got it. Lately, I assume this is sort of bypassing some of checks. I would like to add another check to distinguish it.

I will merge this PR as it's a good reference for me. Notice this is not permanent.
I can't hold all tests in a long run, and need to drop some tests. So, I'd suggest to keep your branch.

btw, to keep the list of various libs instead of this repo, I created https://github.com/dai-shi/lets-compare-global-state-with-react-hooks. It's not very popular yet unfortunately.

**Comment by oleggrishechkin** [comment]
Yes, async notify passed 9 check.

**Comment by dai-shi** [comment]
Right, that's not the original intention of the check. I guess valtio and rx libs pass it because of that too.

**Pull request merged**